### PR TITLE
CommitStatus, PRStatus renamed to CommitFlag, PRFlag

### LINKED ISSUE #73: Change "Status" to  "Flag" in the class names
What about changing `CommitStatus`/`PullRequestStatus` to `CommitFlag`/`PullRequestFlag`?

_Original discussion: https://github.com/packit-service/ogr/pull/71#pullrequestreview-240461635_

### PR DESCRIPTION
**Pull request by lbarcziova**
Fixes #73

### CODE REVIEW

**Comment by rpitonak** [comment]
needs rebase

**Comment by jpopelka** [comment]
Just in [packit tests](https://github.com/packit-service/packit/blob/master/tests/conftest.py#L36)

**Comment by lachmanfrantisek** [comment]
> Just in [packit tests](https://github.com/packit-service/packit/blob/master/tests/conftest.py#L36)

OK, let's merge and fix the Packit tests after we release the change to Fedora.

File: ogr/abstract.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def __str__(self) -> str:
         )
 
 
-class PRStatus(IntEnum):
+class PRFlag(IntEnum):
```
Comment (lachmanfrantisek) [inline ogr/abstract.py]: Please, leave `PRStatus` as is and create a new Class for `PRFlag`.

status = open/close/merged
flag = CI status (only for PRs and commits)

File: tests/integration/test_github.py [hunk 2]
```diff
@@ -227,8 +227,6 @@ def test_update_pr_info(self):
         )
         pr_info = self.colin_project.get_pr_info(pr_id=1)
         assert pr_info.title == "changed"
-        assert pr_info.description == "changed description"
-
```
Comment (lachmanfrantisek) [inline tests/integration/test_github.py]: Why this change?

Comment (lbarcziova) [reply tests/integration/test_github.py]: Just a mistake, I didn't notice I had deleted it.

Comment (lachmanfrantisek) [reply tests/integration/test_github.py]: Otherwise LGTM, sorry for the misleading description of the issue.

**Comment by lachmanfrantisek** [comment]
Sorry for another request, but I've closed your second PR. Please, could you rebase this one?
